test/sot/test_24_exceptions.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2025 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import unittest
+
+from test_case_base import (
+    TestCaseBase,
+)
+
+import paddle
+from paddle.jit.sot.psdb import check_no_breakgraph
+from paddle.jit.sot.utils import strict_mode_guard
+
+NOT_ALLOW_FALLBACK = sys.version_info < (3, 11) and sys.version_info >= (3, 9)
```

Comment:
```suggestion
NOT_ALLOW_FALLBACK = not ALREADY_SUPPORTED_EXCEPTION
```

从 `python/paddle/jit/sot/opcode_translator/executor/opcode_executor.py` import，不要维护两遍